README.md

```diff
@@ -94,12 +94,11 @@ $process->stdout->on('close', function () {
 
 $process->stdin->write($data);
 $process->stdin->end($data = null);
-$process->stdin->close();
 // …
 ```
 
 For more details, see the
-[`DuplexStreamInterface`](https://github.com/reactphp/stream#duplexstreaminterface).
+[`ReadableResourceStream`](https://github.com/reactphp/stream#readablestreaminterface) and [`WritableResourceStream`](https://github.com/reactphp/stream#writablestreaminterface).
```

Comment:
See above comments about `ReadableStreamInterface` and `WritableStreamInterface`.